MalwareAnalyzer/views/domain_check.py

```diff
@@ -147,6 +147,8 @@ def get_domains(urls):
         domains = []
         for url in urls:
             parsed_uri = urlparse(url)
+            if urlparse(url).port:
+                url = urlparse(url).hostname
             if not urlparse(url).scheme:
```

Comment:
`parsed_uri.scheme `

Reply:
if doing this the url will not be parsed after removing the port :( need to rethink it